packages/quantic/scripts/build/deploy-community.ts

```diff
@@ -212,12 +213,15 @@ async function deployCommunity(
 
       success = true;
     } catch (error) {
-      if (retry === 2) {
+      if (retry === 5) {
         throw error;
       }
+      // The deployment may fail because the community is still being created.
+      // Wait for 60 seconds then retry.
```

Comment:
waiting a whole minute seems too long to me. 10 or 20 seconds with more retries would be quicker at finding that moment when the community is ready.

Reply:
All right 👍